app/Http/Requests/StoreProjectContextRequest.php

```diff
@@ -19,6 +19,14 @@ public function rules(): array
         ];
     }
 
+    public function attributes(): array
+    {
+        return [
+            'context' => __('project context'),
+            'ancestor' => __('previous project'),
```

Comment:
Here's another idea, although I don't know which is better really.
```suggestion
            'ancestor' => __('original project'),
```

Reply:
We currently use both terms previous project and original project in other messages, and there are 2 usages of previous project, and one usage of original project. I think we should always be consistent throughout the messages.

Reply:
If they are used for the same purpose, I'd agree that they should be consistent.